src/components/dialogs/create-contingency-list-dialog.js

```diff
@@ -129,7 +149,13 @@ export const CreateContingencyListDialog = ({ open, onClose }) => {
     };
 
     const handleChangeContingencyListType = (event) => {
-        setContingencyListType(event.target.value);
+        if (isUnsavedChanges) {
+            setOpenConfirmationPopup(true);
+            setChosenContingencyListType(event.target.value);
+        } else {
+            handlePopupConfirmation();
```

Comment:
We call `handlePopupConfirmation` without openning the popup at all.
Rename the function, or cut it in multiple functions but this is strange.
(I know this code already exists for filter dialog, maybe in another PR ?)